packages/core/src/lib/TwaManifest.ts

```diff
@@ -473,6 +486,8 @@ export class TwaManifest {
           webManifest['name']?.substring(0, SHORT_NAME_MAX_SIZE)),
       display: this.getNewFieldValue('display', fieldsToIgnore, oldTwaManifest.display,
           asDisplayMode(webManifest['display']!)!),
+      scope: this.getNewFieldValue('scope', fieldsToIgnore, oldTwaManifest.scope,
```

Comment:
@andreban Please check this carefully, because I don't understand fully what's happening here.

Reply:
Looks right to me - this if `scope` is in `fieldsToIgnore`, it will ignore the new value, basically.